examples/Basic/package.json

```diff
@@ -4,19 +4,42 @@
 	"private": true,
 	"scripts": {
 		"start": "node node_modules/react-native/local-cli/cli.js start",
-		"test": "jest"
+		"test": "jest",
+		"postinstall": "rm -rf node_modules/react-native-blur/.git node_modules/react-native-blur/examples/",
```

Comment:
We have discussion dedicated to this feature [here](https://github.com/react-native-community/react-native-blur/pull/191)